casual/casual-caller/src/main/java/se/laz/casual/connection/caller/config/Configuration.java

```diff
@@ -141,34 +165,41 @@ public static class Builder
         private Integer validationIntervalMillis;
         private Boolean transactionStickyEnabled;
         private Long topologyChangeDelayMillis;
+        private String routeFilename;
 
         public Configuration build()
         {
             return new Configuration(this);
         }
 
-        public Builder jndiSearchRoot(String jndiSearchRoot)
+        public Builder withJndiSearchRoot(String jndiSearchRoot)
```

Comment:
why the change to `wither`?

Reply:
2 reasons, 1 that it is what we are using in most if not all other builders.
Secondly, groovy got confused with function/attribute name being exactly the same.